tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -87,9 +138,54 @@ def modify_feed(test_values, get_configuration, request):
     vd.clean_vuln_and_sys_programs_tables()
 
 
-def test_no_feed_changes(copy_files, clean_vuln_tables, after_execution_clean_vuln_tables_function,  get_configuration, configure_environment, daemons_handler,
-                         file_monitoring):
-    """Check if the feed is imported successfully by default."""
+def test_no_feed_changes(copy_files, clean_vuln_tables, after_execution_clean_vuln_tables_function,  get_configuration,
+                         configure_environment, daemons_handler, file_monitoring):
+    '''
+    description: Check if the feed is imported successfully by default. To do this, a vulnerability event is expected
+                 and check if the number of vulnerabilities inserted in the VULNERABILITIES table of CVE DB is the
+                 expected.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - copy_files:
+            type: fixture
+            brief: Write dict data to JSON file.
+        - clean_vuln_tables:
+            type: fixture
+            brief: Clean vulnerabilities tables.
+        - after_execution_clean_vuln_tables_function:
+            type: fixture
+            brief: Clean vulnerabilities tables after execution.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - daemons_handler:
+            type: fixture
+            brief: Handler of Wazuh daemons.
+        - file_monitoring:
+            type: fixture
+            brief: Handle the monitoring of a specified file.
+
+    assertions:
+        - Assert that the number of vulnerabilities is equal to the expected number of vulnerabilities inserted in the
+          VULNERABILITIES table of CVE DB.
+
+    input_description:
+        - Test cases are defined in the custom_alas_feed.json and custom_alas2_feed.json files. Those files contain two
+          different kinds of vulnerability.
```

Comment:
I would also mention the configuration from `data/configuration/wazuh_invalid_alas_feed.yaml`

```suggestion
        - Test cases are defined in the custom_alas_feed.json and custom_alas2_feed.json files. Those files contain two
          different kinds of vulnerabilities.
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/c02bcd31ed6d3d0f57d519bdc2051b8eafeede60#:~:text=c02bcd31ed6d3d0f57d519bdc2051b8eafeede60)